### ISSUE TITLE
[Accessibility]MAS 4.2.1: (eclipse/edit '') The Name property of a focusable element must not be null.

### ISSUE THREAD

**Issue by Manisha370** [microsoft/azure-tools-for-java#3611]
The following accessibility issue needs investigation.

**App:** eclipse

GitHubTags: #A11yTCS; #A11yMAS; #A11yAuto; #AI4D; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-NewFeature; #MAS4.2.1; #C+E-H; #JavatoolingforIntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:**
OS: Windows (Version 1903, Build: 18362.356)

**Element path:** edit ''

**Issue Details:** The Name property of a focusable element must not be null. [Section 508 502.3.1](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology)

**How To Fix:** Provide a UI Automation Name property that concisely identifies the element.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}

![Keros_Name property of a focusable element must not be null](https://user-images.githubusercontent.com/51435937/65500391-dd63fe00-dedc-11e9-9431-ca6de8d4d6dd.png)

**Comment by t-rufang** [comment]
Checked our code, this dialog is extended from class `org.eclipse.jface.dialogs.TitleAreaDialog`, which is provided by Eclipse. Therefore, it's out of the scope of our toolkit.
Please also find another similar issue when we open a Eclipse updater dialog (Help -> Perform Serup Tasks...)
![image](https://user-images.githubusercontent.com/32627233/68462730-19aeac00-0248-11ea-81d1-a6dbb16287c2.png)

![image](https://user-images.githubusercontent.com/32627233/68462506-a7d66280-0247-11ea-82de-354cd57ba1a1.png)

![image](https://user-images.githubusercontent.com/32627233/68462503-a3aa4500-0247-11ea-8e7c-7ef9740205d1.png)

**Comment by t-rufang** [comment]
- Importance of experience: Low, when user focus to sub-title textfield or help button, screen reader user doesn't know the name of current focused element.
- Quality of experience: Little, the text of the component itself will tell user the meaning of current focused element
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Tanushagottipati** [comment]
Closing this bug as approved by SME

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749711
